src/test/java/com/ge/predix/uaa/token/lib/FastTokenServiceTest.java

```diff
@@ -1,303 +0,0 @@
-import org.springframework.http.HttpMethod;
-import org.springframework.security.jwt.crypto.sign.InvalidSignatureException;
-import org.springframework.security.oauth2.common.exceptions.InvalidTokenException;
-import org.springframework.security.oauth2.provider.OAuth2Authentication;
-import org.springframework.web.client.RestClientException;
-import org.springframework.web.client.RestTemplate;
-import org.testng.annotations.Test;
-
-import com.fasterxml.jackson.core.type.TypeReference;
-import com.fasterxml.jackson.databind.ObjectMapper;
-
-public class FastTokenServiceTest {
-
-    private final TestTokenUtil testTokenUtil = new TestTokenUtil();
-
-    private static final String TOKEN_ISSUER_ID = "http://localhost:8080/uaa/oauth/token";
-
-    private static final String TOKEN_KEY_URL = "https://localhost:8080/uaa/token_key";
-
-    private final FastTokenServices services;
-
-    public FastTokenServiceTest() throws Exception {
-        this.services = new FastTokenServices();
-        this.services.setRestTemplate(mockRestTemplate(false));
-        this.services.setTrustedIssuers(trustedIssuers());
-        this.services.afterPropertiesSet();
-    }
-
-    private List<String> trustedIssuers() {
-        List<String> trustedIssuers = new ArrayList<>();
-        trustedIssuers.add(TOKEN_ISSUER_ID);
-        return trustedIssuers;
-    }
-
-    private RestTemplate mockRestTemplate(final boolean updatedSigningKey) {
-        ParameterizedTypeReference<Map<String, Object>> typeRef =
-                new ParameterizedTypeReference<Map<String, Object>>() {
-            // Nothing to add.
-        };
-
-        RestTemplate restTemplate = Mockito.mock(RestTemplate.class);
-        if(updatedSigningKey) {
-            Mockito.when(restTemplate.exchange(TOKEN_KEY_URL, HttpMethod.GET, null, typeRef))
-                    .thenReturn(TestTokenUtil.mockUpdatedTokenKeyResponseEntity());
-        } else {
-            Mockito.when(restTemplate.exchange(TOKEN_KEY_URL, HttpMethod.GET, null, typeRef))
-                    .thenReturn(TestTokenUtil.mockTokenKeyResponseEntity());
-        }
-        return restTemplate;
-    }
-
-    @Test
-    public void testLoadAuthentication() throws Exception {
-        String accessToken = this.testTokenUtil.mockAccessToken(60);
-        OAuth2Authentication result = this.services.loadAuthentication(accessToken);
-        assertNotNull(result);
-        assertEquals("cf", result.getOAuth2Request().getClientId());
-        assertEquals("marissa", result.getUserAuthentication().getName());
-        assertEquals("1adc931e-d65f-4357-b90d-dd4131b8749a",
-                ((RemoteUserAuthentication) result.getUserAuthentication()).getId());
-        assertNotNull(result.getOAuth2Request().getRequestParameters());
-        assertNull(result.getOAuth2Request().getRequestParameters().get(Claims.ISS));
-    }
-
-    @Test
-    public void testLoadAuthenticationLoadingCacheExpiry() throws Exception {
```

Comment:
can we add missing tests from this class to src/test/java/com/ge/predix/uaa/token/lib/FastTokenServicesTest.java ?

Reply:
Added this case